app/styles/component/_article-media-thumbnail.scss

```diff
@@ -33,6 +33,7 @@
 		}
 	}
 
+	&.downloadable figcaption,
 	&.is-ogg figcaption {
```

Comment:
looks like this class has been removed?

Reply:
@hakubo it is used in article-content, comes from app